bazel/README.md

```diff
@@ -561,7 +561,8 @@ You may use any [Remote Caching](https://docs.bazel.build/versions/master/remote
 as an alternative to this.
 
 This requires Go 1.11+, follow the [instructions](https://golang.org/doc/install#install) to install
-if you don't have one.
+if you don't have one. The command must run with Go modules enabled, and should work when run from
```

Comment:
> The command must run with Go modules enabled

I don't really think so? It can run with GO111MODULE=off as well?

Reply:
hmm I'm not familiar enough with the go toolchain to know what's going on. Is the issue that certain paths will make it attempt to use sources from GOPATH? And in those cases using go modules fixes it because go modules don't rely on GOPATH? 

In any case I'll just leave out the explanation (since I'm apparently no good at understanding Go tooling lol).